src/api/Base.service.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable */
+import "whatwg-fetch";

```

Comment:
why are we importing `whatwg-fetch` when there is a build-in fetch in react-native?